src/testcases.txt

```diff
@@ -0,0 +1,9 @@
+9
+2
+1,5,7,2,6,9,1,4,2
+1,2
+
+9
+2
+1,5,7,2,6,9,1,4,2
+2,1
```

Comment:
There should be a different file for testing.
Inside this test file, you should store the answers too. Then obtain those answers from the algorithm. Then compare your obtained answers to the saved answers and print whether successful or not.